File: Aman Gautam/VideoConferencingApp/app/src/main/java/com/example/videoconferencingapp/MainActivity.java
package com.example.videoconferencingapp;

import androidx.appcompat.app.AppCompatActivity;
import android.view.View;

import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;

import org.jitsi.meet.sdk.JitsiMeetActivity;
import org.jitsi.meet.sdk.JitsiMeetConferenceOptions;

import java.net.MalformedURLException;
import java.net.URL;


public class MainActivity extends AppCompatActivity {
    EditText t1;
    Button b1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        //  To handle exceptions
        try {
            JitsiMeetConferenceOptions options = new JitsiMeetConferenceOptions.Builder()
                    .setServerURL(new URL(""))
                    .setWelcomePageEnabled(false)
                    .build();

        } catch (MalformedURLException e) {
            e.printStackTrace();
        }
    }
        public void onClick(View v){
            t1 = findViewById(R.id.room_name);
            String text = t1.getText().toString();

            if(text.length() > 0){
                JitsiMeetConferenceOptions options = new JitsiMeetConferenceOptions.Builder()
                        .setRoom(text)
                        .setWelcomePageEnabled(true)
                        .build();
                JitsiMeetActivity.launch(this, options);
            }

        }


}

File: memes_android_app_kotlin/app/src/main/java/com/example/elitememes/MainActivity.kt
package com.example.elitememes

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.ImageView
import com.android.volley.Request
import com.android.volley.toolbox.StringRequest
import com.android.volley.toolbox.Volley
import com.bumptech.glide.Glide
import org.json.JSONObject


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun nextMeme(view: View) {
        val api: String = "https://meme-api.herokuapp.com/gimme"
        val imgView = findViewById<ImageView>(R.id.imageView)

        val queue = Volley.newRequestQueue(this)

// Request a string response from the provided URL.
        val stringRequest = StringRequest(
            Request.Method.GET,
            api,
            { response: String ->
                // Display the first 500 characters of the response string.
                val obj = JSONObject(response)
                val img = obj.getString("url")
                Glide.with(this).load(img).into(imgView)
            },
            { println("That didn't work!") }
        )
// Add the request to the RequestQueue.
        queue.add(stringRequest)

    }


}